pkg/providers/instancetype/instancetypes.go

```diff
@@ -269,6 +269,8 @@ func (p *DefaultProvider) getInstanceTypes(ctx context.Context) (map[string]*ske
 		return nil, fmt.Errorf("fetching SKUs using skewer, %w", err)
 	}
 
+	// TODO when https://github.com/Azure/skewer/pull/24 merges
+	// skus := cache.List(ctx, skewer.Includes(getKarpenterWorkingSKUs()))
```

Comment:
Something like this would work; the filter needs to have `FilterFn` signature; left a comment on https://github.com/Azure/skewer/pull/24

Reply:
Addressed in https://github.com/Azure/skewer/pull/24, thanks!